src/VscProcess.cpp

```diff
@@ -486,3 +502,27 @@ void VscProcess::readFromVehicle()
     lastRemoteStatusRxTime = ros::Time::now();   
   }
 }
+
+void VscProcess::settingsGrab()
```

Comment:
I prefer grabSettings or better readSettings() but doesn't really matter.

Reply:
I would also vote for readSettings(). That's more consistent with function names across our code base.

Reply:
changed to `readSettings()`.